test/unit/Utils.spec.ts

```diff
@@ -0,0 +1,80 @@
+import { expect } from 'chai';
+import os from 'os';
+import * as utils from '../../lib/Utils';
+import { PairFactory } from '../../lib/consts/Database';
+
+describe('utils', () => {
```

Comment:
`utils` should be capitalized
```suggestion
describe('Utils', () => {
```